lib/src/components/arm/service.dart

```diff
@@ -6,6 +6,7 @@ import '../../resource/manager.dart';
 import '../../utils.dart';
 import 'arm.dart';
 
+/// {@category Components}
```

Comment:
All of the category definitions in all of the `service.dart` files are actually unneeded since we don't export these files, so no documentation exists for these

Reply:
(No need to do anything, just commenting here for documentation)